packages/@okta/vuepress-site/docs/guides/email-customization/customize-email-templates/index.md

```diff
@@ -117,9 +117,18 @@ You can also reference these org-level attributes, such as:
 
 All conditional logic that is supported by the Velocity Templating Engine, such as `if`, `elseif`, or `else` constructs and `foreach` loops, is available for you to use in your templates. See the [Velocity documentation](http://velocity.apache.org/engine/1.7/user-guide.html).
 
-#### Email templates for customization
+<ApiLifecycle access="ie" /><br>
+
+#### Access to app context
+
+Okta Identity Engine orgs can now call app context within emails via Velocity Templating Language. When an end user enters an authentication flow, OIE stores that app context in the state token, which can then be called using the following app macros:
```

Comment:
Suggest tweaking the wording just a bit:

"Okta Identity Engine orgs now have access to app context within emails using the Velocity Templating Language. When an end user enters an authentication flow, OIE stores the app context in the state token. The following properties are available on the app context:"

Reply:
thanks done